package.json

```diff
@@ -30,15 +30,19 @@
     "clean": "lerna clean --yes && rm -rf node_modules",
     "clean:install": "npm run clean && npm install",
     "jest": "jest",
-    "wdio": "node ./packages/terra-functional-testing/bin/test-runner.js",
     "lint": "npm run lint:js",
     "lint:js": "eslint --ext .js,.jsx .",
+    "pack": "webpack --config webpack.config.js -p",
```

Comment:
webpack will pick up this file by default so you wouldn't have to pass in `--config webpack.config.js`

Reply:
Removed:

https://github.com/cerner/terra-toolkit/pull/417/commits/7b43d2d93a6c308d7990995e74934d87b098aace